packages/jest-stylelint-runner/package.json

```diff
@@ -20,6 +20,7 @@
   },
   "files": [
     "package.json",
+    "run.js",
```

Comment:
question: do we need to include `index.js` here as well?

Reply:
Yeap.

Reply:
Wait, I thought we put the files into a `src` folder?

Reply:
No, one of your comments was to move it out of a src folder, no? 🤔

Reply:
Should I move them back into a src folder?

Reply:
I meant the tests to move them out of the src folder. Anyway, we can do that as a follow up...